### DISCUSSION TITLE
Should we delete the Profile Link Feature?

### DISCUSSION THREAD

**Discussion by jellypuno** [zowe/zowe-explorer-vscode#1110] [Q&A]
I'm not sure if anybody is using this but maybe we should prioritize removing this feature.

**Comment by zFernand0** [comment]
Curious if @Joe-Winchester or @phaumer know someone interested in this sort of feature.
But I agree since base profile serve a similar purpose! 👍

**Reply by Joe-Winchester** [reply]
> Curious if @Joe-Winchester or @phaumer know someone interested in this sort of feature.
> But I agree since base profile serve a similar purpose! 👍

I recall Colin Stone discussing this feature a while back to allow folks who had a TSO profile and a z/OSMF one (and possibly an SSH) to link them together.   I know one user in IBM who uses this but they're not on github so I'll e-mail them the issue and try to get their input.

**Reply by ghost** [reply]
Joe contacted me yesterday as I was discussed it with Colin Stone about a year ago. Let me report that I no longer need for this feature.

**Comment by jellypuno** [comment]
Issue #1131 is opened to remove this feature. Thank you everyone.
